### PR TITLE
doc fixes and updates

### PR DESCRIPTION
**Pull request by ned1313**
# Description

Update the Google extauth guide 

# Context

This is in response to the issue #3557 
BOT NOTES: 
resolves https://github.com/solo-io/gloo/issues/3557

### CODE REVIEW

File: docs/content/guides/security/auth/extauth/oauth/google/_index.md [hunk 1]
```diff
@@ -150,6 +148,17 @@ spec:
...
+        single:
+          kube:
+            ref:
+              name: petclinic
+              namespace: default
+            port: 80
+        options:
+          prefixRewrite: '/login'
     - matchers:
       - prefix: /
```
Comment (kdorosh) [inline docs/content/guides/security/auth/extauth/oauth/google/_index.md:163]: can we do the example without using any `/` matchers to be very clear?

File: docs/content/guides/security/auth/extauth/oauth/google/_index.md [hunk 2]
```diff
@@ -167,11 +176,11 @@ spec:
 {{< /highlight >}}
 
 {{% notice note %}}
-Note this is a simplistic example that has a `/` catch-all path prefix route. Gloo needs to handle the `/callback` route and does so in the External Auth service. If you don't have a matching route (ie, either using a `/` or `/callback`) for the `callback` setting, you'll see `404`s when the Identity Provider tries to callback to Gloo with the correct tokens. Please reach out to us on the [Slack](https://slack.solo.io) if you run into trouble here.
+This example is sending the `/callback` prefix to `/login`, a path that does not exist. The request will not be interpreted by the petclinic service, but you could easily add code for the `/login` path that would parse the state information from Google and use it to load a profile of the user.
```
Comment (kdorosh) [inline docs/content/guides/security/auth/extauth/oauth/google/_index.md:179]: lets send this to another page within the petclinic that exists but is clearly different

File: docs/content/guides/security/auth/extauth/oauth/google/_index.md (hunk 1 above)
Comment (ned1313) [reply docs/content/guides/security/auth/extauth/oauth/google/_index.md]: Got it working, update pushed.

**Review by kdorosh — CHANGES_REQUESTED** [review]
let's add a non user facing changelog (eg https://github.com/solo-io/gloo/blob/master/changelog/v1.5.0-beta25/no-cleanup-gloo-in-ci.yaml) that references https://github.com/solo-io/gloo/issues/3557

File: docs/content/guides/security/auth/extauth/oauth/google/_index.md (hunk 2 above)
Comment (ned1313) [reply docs/content/guides/security/auth/extauth/oauth/google/_index.md:183]: If an existing page is used, the application appears to send us back to the main `/app` page.

**Comment by ned1313** [comment]
> 
> 
> let's add a non user facing changelog (eg https://github.com/solo-io/gloo/blob/master/changelog/v1.5.0-beta25/no-cleanup-gloo-in-ci.yaml) that references #3557

Should this go in v1.5.0-beta26?

**Comment by kdorosh** [comment]
https://github.com/solo-io/gloo/pull/3660#issuecomment-697856947

@ned1313 

https://github.com/solo-io/gloo/releases/tag/v1.5.0-beta25 is the latest release, so yes

File: docs/content/guides/security/auth/extauth/oauth/google/_index.md (hunk 2 above)
Comment (kdorosh) [reply docs/content/guides/security/auth/extauth/oauth/google/_index.md:183]: ok we can go back to the main page, that's fine

**Review by kdorosh — APPROVED** [review]

**Pull request merged**